reference/7.2/Microsoft.PowerShell.Core/About/about_Logging_Non-Windows.md

```diff
@@ -279,52 +280,20 @@ log file named `powershell.log`.
 ## Viewing PowerShell log data on macOS
 
 PowerShell logs to Apple's unified logging system, a feature of macOS that
-allows for the collection and storage of system and application logs in a single
-centralized location.
-
-Apple's unified logging system stores log messages in binary format. Use the
-Console app or log tool to query the unified logging system for PowerShell
-entries.
-
-### Viewing PowerShell log data in the Console application on macOS
-
-The **Console** application on macOS is a utility that provides a graphical user
-interface for viewing log data. The **Console** application is included with
-macOS by default and can be accessed by opening the **Utilities** folder in the
-**Applications** folder.
-
-Use the following steps to view PowerShell log data in the Console application
-on macOS:
-
-1. Search for the **Console** application and launch it.
-1. Select the Machine name under **Devices**.
-1. In the **Search** field, enter `pwsh` for the PowerShell main binary and
-   press <kbd>return</kbd>.
-1. Change the search filter from `Any` to `Process`.
-1. Click **Start**.
-1. Run `pwsh` to generate PowerShell information to log.
-
-The process ID for a running instance of PowerShell is stored in the `$PID`
-variable. Use the following steps to filter on a specific process instance of
-PowerShell in the **Console** application.
-
-1. Run an instance of `pwsh`.
-1. Run `$PID` in the instance of PowerShell started in the previous step to
-   determine its process ID.
-1. Enter the process ID for `pwsh` in the **Search** field and press
-   <kbd>return</kbd>.
-1. Change the search filter from `Any` to `PID`.
-1. Click **Start**.
-1. Generate PowerShell information to log from the instance of PowerShell
-   started in the first step.
+allows for the collection and storage of system and application logs in a
+single centralized location.
 
-For more information, see [view log messages in Console on Mac][08].
+Apple's unified logging system stores log messages in binary format. You must
+use the `log` tool to query the unified logging system for PowerShell log
+events. The PowerShell log events don't appear in the **Console** application
+on macOS. Console app is designed for the older _syslog-based_ logging that
+predates the unified logging system.
 
 ### Viewing PowerShell log data from the command line on macOS
 
 To view PowerShell log data from a command line on macOS, use the `log` command
 in the **Terminal** or other shell host application. These commands can be run
-from **PowerShell**, **Z shell** (**Zsh**), or **Bash**.
+from **PowerShell**, **ZShell**, or **Bash**.
```

Comment:
According to Apple, it's `zsh` (Z shell) [support.apple.com/102360](https://support.apple.com/102360).